src/bootrom.rs

```diff
@@ -0,0 +1,79 @@
+use core::mem;
+
+// A bootrom function table code.
+pub type RomFnTableCode = [u8; 4];
```

Comment:
Table codes are 2 chars, not 4.

Reply:
The code themselves are 2 chars but if you take a look at macro `rom_table_code` 'bootrom.h' in the SDK you will see that it's actually 32 bit (: